### ISSUE TITLE
How to calculate OoD Store?

### ISSUE THREAD

**Issue by xiongminghao** [thu-vis/OoDAnalyzer#2]
Is the score calculated in advance?

**Comment by chencjGene** [comment]
Yes. This repo does not include the codes to calculate OoD scores. You can use other OoD detection method, like deep ensemble, to calculate OoD scores.  We will open source the codes for OoD detection when we have time to clean them.

**Comment by anastasia-spb** [comment]
Hello, is it possible to try OoD Score algorithm? If the code is not ready to be published, perhaps it is possible to put it into Google drive alongside with "Animals.zip" file?

Could you please also comment on how embedding for each image in provided example dataset was calculated? The one, which is stored in ``embed_X.pkl`` file.
